### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by v1rtl** [talentlessguy/tinyhttp#217] [General]
This is the main discussion for all feedback for tinyhttp, both positive and negative.

Feel free to share your thoughts about the framework here.

**Comment by sahitono** [comment]
Hi, thank you for making this great framework. I've been using it for my small project, and it's working fine.
Due to its simplicity, it is easier to modify and add adjustments. So far, I've replicated the swagger plugin and added some features, such as support for sub-apps routes and adding more fields in the schema. If you'd like to see it, I can create a PR

**Reply by aarontravass** [reply]
Hi @sahitono thanks for the feedback. Do go ahead and create a PR

**Reply by v1rtl** [reply]
hi, thanks for the feedback!

tinyhttp supports subapp routes already, you don't have to implement them

there's also `@tinyhttp/swagger` for Swagger integration

I'm interested what you mean by schema, do you mean validation?

**Reply by sahitono** [reply]
Sorry, I think there's a misunderstanding here. So,  what I meant is that sub-apps support @tinyhttp/swagger. 
For schema, it's for the swagger. So,  I added another field, such as description and security.

**Reply by sahitono** [reply]
For validation, I created middleware with joi

**Comment by calumk** [comment]
Hey - The new docs website is Great 👍

**Reply by v1rtl** [reply]
thank you very much!
